lua/music-controls/init.lua

```diff
@@ -18,26 +18,25 @@ end
 -- If not, it will display a notification.
 local function check_playerctl_installed()
     if vim.fn.executable('playerctl') == 0 then
-        vim.notify('playerctl is not installed', 'error')
+        notify('playerctl is not installed', 'error')
         return false
     end
 
     return true
 end
 
--- Execute the 'playerctl' command.
+-- Execute the 'playerctl' command and get it's output.
 local function exec_command(cmd)
-    -- Execute the command and get the output.
-    local handle = io.popen(cmd)
-
-    if handle == nil then
-        return nil
+    local output = vim.fn.systemlist(cmd)
+    local result = ''
+
+    if output then
+      for _, line in ipairs(output) do
+      remove_newline(line)
+      result = line
+      end
     end
```

Comment:
Please fix the indentation here.

```suggestion
    if output then
        for _, line in ipairs(output) do
            remove_newline(line)
            result = line
        end
    end
```

Reply:
Done in my lastest commit.